Joysticks/mfjoystick.schema.json

```diff
@@ -59,28 +59,28 @@
         "$id": "#root/Outputs/items",
         "title": "Items",
         "type": "object",
-        "required": ["Name", "Label", "Byte", "Bit"],
+        "required": ["Label", "Byte", "Bit"],
```

Comment:
There's a broader problem here that still needs to be addressed. The schema defines four properties on an output: Label, Id, Byte, and Bit.

The code expects: Label, Name, Byte, Bit.

The JSON files we're using currently have: Label, Id, Byte, Bit.

What's it supposed to be?

Reply:
i think ID is better

Reply:
I looked at this in more detail. The code that uses this expects a "Name" property:

```C#
Definition?.Outputs?.ForEach(output => Lights.Add(new JoystickOutputDevice() { Label = output.Label, Name = output.Name, Byte = output.Byte, Bit = output.Bit }));
``

Comparing this to the JoystickInput definition, it appears it uses `Id` but maps that to a name based on the device type via a function. To align the two I believe the schema and definition file should expose an "Id" property, and the Name should be a function like JoystickInput that simply returns the Id.